recipes/boost/all/conanfile.py

```diff
@@ -124,6 +126,8 @@ class BoostConan(ConanFile):
         "extra_b2_flags": "None",
         "i18n_backend": "iconv",
         "visibility": "hidden",
+        "addr2line_location": "/usr/bin/addr2line",
```

Comment:
how exactly does it work? does it hardcode addr2line location into the libraries?
if so, how is it expected to work if addr2line located in different place on the consumer's machine than on CI machine?

Reply:
I ran `strings libboost_stacktrace_addr2line.a` and did not find an occurrence of `/usr/bin/addr2line`.
So I think the library only contains the classes and I think it is possible to override the location of `addr2line` when compiling by defining some environment variable.